samples/client/petstore/typescript-axios/builds/composed-schemas/api.ts

```diff
@@ -132,7 +132,7 @@ export interface InlineObject {
      * @type {any}
      * @memberof InlineObject
      */
-    file?: any;
+    file?: any | null;
```

Comment:
are changes in typescript intentional?

Reply:
Does any allow null?

Reply:
It looks like any does allow in null per: https://dev.to/aumayeung/typescript-data-types-null-void-undefined-never-and-object-432h#:~:text=We%20can%20only%20assign%20null,type%20and%20the%20null%20type.
Most typescript changes have been reverted with my recent update. Changing them was not intentional.

Reply:
Updated typescript-axios and added updatePropertyForAnyType which ensures that this file stays the same